O/OpenBLAS/common.jl

```diff
@@ -182,6 +182,8 @@ function openblas_script(;num_64bit_threads::Integer=32, openblas32::Bool=false,
         flags+=(TARGET=ARMV7)
     elif [[ ${target} == powerpc64le-* ]]; then
         flags+=(TARGET=POWER8 DYNAMIC_ARCH=1)
+    elif [[ ${target} == riscv64-* ]]; then
+        flags+=(TARGET=RISCV64_GENERIC)
```

Comment:
We should do dynamic arch as well, if available

Reply:
Ok.